controllers/applicationvolumereplication_controller.go

```diff
@@ -417,9 +424,60 @@ func (a *AVRInstance) processPausedSubscription(
 		return IsManifestInAppliedState(pvMW)
 	}
 
+	// TODO: remove the following calls when @BenamarMk's changes go in: getVRGFromManagedCluster, isVRGReadyForFailback
+	vrg, err := a.reconciler.getVRGFromManagedCluster(subscription.Name, subscription.Namespace, newHomeCluster)
```

Comment:
@BenamarMk acking and merging with the understanding that PR #69 would refactor this part as suitable for its consumption.